packages/block-editor/src/components/iframe/use-scale-canvas.js

```diff
@@ -409,20 +433,24 @@ export function useScaleCanvas( {
 				// the iframe at this point when we're about to animate the zoom out.
 				// The iframe scrollTop, scrollHeight, and clientHeight will all be
 				// the most accurate.
-				transitionFromRef.current.containerHeight =
-					transitionFromRef.current.containerHeight ??
-					containerHeight; // Use containerHeight, as it's the previous container height value if none was set.
```

Comment:
Using the `transitionFromRef.current.containerHeight` won't be as accurate, as it could have changed from a manual resize between animations. Relying on `containerHeight` will be more accurate.